ui/src/components/Document/utils.js

```diff
@@ -1,8 +1,6 @@
-// NOTE: random function to get different covers for books.
 
+// NOTE: random function to get different covers for books.
 export function getCover(edition = '42') {
   const imageId = 46276847 + parseInt(edition);
   return `https://assets.thalia.media/img/${imageId}-00-00.jpg`;
```

Comment:
why not moving this to backend instead? Just double check if `thalia.media` can be used freely.

Reply:
This will be removed completely once I address couple of issues with BookCard